src/core/public/chrome/ui/header/header.tsx

```diff
@@ -167,7 +166,7 @@ export class Header extends Component<HeaderProps, State> {
         aria-label={i18n.translate('core.ui.chrome.headerGlobalNav.toggleSideNavAriaLabel', {
           defaultMessage: 'Toggle side navigation',
         })}
-        onClick={() => this.navDrawerRef.current.toggleOpen()}
+        onClick={() => this.navDrawerRef.current && this.navDrawerRef.current.toggleOpen()}
```

Comment:
What caused the `nafDrawerRef.current` to now be potentially null in the handler?

also NIT: `this.navDrawerRef.current && this.navDrawerRef.current.toggleOpen()` => `this.navDrawerRef.current?.toggleOpen()`

Reply:
It was always potentially `null` but EUI didn't provide types for `EuiNavDrawer` until now. 
As such, `NavDrawer` was using the generic `HTMLElement` to type the `ref` and not `EuiNavDrawer`

++ for `this.navDrawerRef.current?.toggleOpen()`
I'll update

Reply:
Updated with optional chaining